> `2026-04-15 16:36:18 - INFO - | make[3]: *** No rule to make target 'arch/arm64/boot/dts/qcom/qcs6490-rb3gen2-staging.dtbo'. Stop. `


This error should go away now that https://github.com/qualcomm-linux/meta-qcom/pull/1956 is merged.

--- a/conf/machine/include/fit-dtb-compatible.inc
+++ b/conf/machine/include/fit-dtb-compatible.inc
@@ -32,6 +32,10 @@ FIT_DTB_COMPATIBLE[lemans-evk+lemans-evk-ifp-mezzanine] = " \
     qcom,qcs9075-iot-subtype1 \
     qcom,qcs9075v2-iot-subtype1 \
 "
+FIT_DTB_COMPATIBLE[lemans-evk+lemans-evk-ifp-mezzanine+lemans-evk-staging] = " \
+    qcom,qcs9075-iot-subtype1-staging \
+    qcom,qcs9075v2-iot-subtype1-staging \
+"
 FIT_DTB_COMPATIBLE[monaco-evk+monaco-evk-camera-imx577] = "qcom,qcs8275-iot"
 FIT_DTB_COMPATIBLE[monaco-evk+monaco-evk-camx] = "qcom,qcs8275-iot-camx"
 FIT_DTB_COMPATIBLE[monaco-evk+monaco-evk-ifp-mezzanine] = "qcom,qcs8275-iot-subtype3"

--- a/conf/machine/iq-8275-evk.conf
+++ b/conf/machine/iq-8275-evk.conf
@@ -15,6 +15,7 @@ KERNEL_DEVICETREE ?= " \
 LINUX_QCOM_KERNEL_DEVICETREE ?= " \
                       qcom/monaco-evk-camx.dtbo \
                       qcom/monaco-evk-ifp-mezzanine.dtbo \
+                      qcom/monaco-evk-staging.dtbo \
                       "
 
 MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " \

--- a/conf/machine/iq-9075-evk.conf
+++ b/conf/machine/iq-9075-evk.conf
@@ -16,6 +16,7 @@ LINUX_QCOM_KERNEL_DEVICETREE ?= " \
                       qcom/lemans-el2.dtbo \
                       qcom/lemans-evk-camx.dtbo \
                       qcom/lemans-evk-ifp-mezzanine.dtbo \
+                      qcom/lemans-evk-staging.dtbo \
                       "
 
 MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " \

